README.md

```
@@ -134,53 +189,87 @@ For instance, this file should work for Sidekiq:
 ![Settings & census configuration](docs/census_conf.png)
 
 - The election module can be configured with the built-in census:
 
 ![Action Delegator census in elections](docs/elections_census.png)
 
-- In elections, results are expanded to details ponderations (if exist):
+- In elections, results are expanded with ponderation breakdowns (if configured):
 
 ![Expanded results as sum of weights](docs/sum_of_weights.png)
 
+#### Architecture Overview
```

markdownlint-cli2 (0.22.1)

[warning] Heading levels should only increment by one level at a time
Expected: h3; Actual: h4

(MD001, heading-increment)